### DISCUSSION TITLE
podman logs processing

### DISCUSSION THREAD

**Discussion by yosiasz** [containers/podman#24779] [General]
Hello,

where are the log files for a container not started as root located? I can see tail with `podman logs -t container_id` but not sure where these are located at if they are saved at all

**Comment by Luap99** [comment]
If you run  `podman inspect` on the container you see the log config which shows the file path when the k8s-file logger is used. However the general default is using journald when available in which case all logs are in the journal and there is no single file for them.

**Comment by yosiasz** [comment]
Ah got it thanks! So when using podman compose I need to configure things so that logs are actually persisted. I think I can even specify the path. Thanks!!
